pkg/cloud/services/networking/port.go

```diff
@@ -161,6 +161,10 @@ func (s *Service) GetOrCreatePort(eventObject runtime.Object, clusterName string
 		record.Warnf(eventObject, "FailedCreatePort", "Failed to create port %s: %v", portName, err)
 		return nil, err
 	}
+	if err = s.replaceAllAttributesTags(eventObject, port.ID, tags); err != nil {
```

Comment:
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1016/files#diff-787f1b361511084b9cd8e6918dfb552589bc1698aa1aa13c9cff9fafde0377aaR131-R142

How are we creating ports with tags if we don't add it here? ^

Reply:
Also `replaceAllAttributesTags` was done for Trunks, not Ports (see the implementation).

Reply:
Thanks for the review, I extended the replaceAllAttributesTags function to support ports and tags.